cartridge_account/src/lib.cairo

```diff
@@ -229,29 +237,15 @@ mod Account {
             self.emit(OwnerAdded { new_owner_guid: new_public_key });
         }
 
-        fn _is_valid_signature(
+        fn _is_valid_ecdsa_signature(
```

Comment:
Leading underscore may be removed as commented in earlier PR.

Reply:
Sorry, I got confused with the branches :). This is fixed in the subsequent PR: [feature/2/5](https://github.com/neotheprogramist/cairo-webauthn/pull/28)